dipy/data/fetcher.py

```diff
@@ -1691,7 +1691,7 @@ def get_fnames(*, name="small_64D"):
 
 
 def read_qtdMRI_test_retest_2subjects():
-    """Load test-retest qt-dMRI acquisitions of two C57Bl6 mice. These
+    r"""Load test-retest qt-dMRI acquisitions of two C57Bl6 mice. These
```

Comment:
same here

Reply:
This one has a backslash in
https://github.com/dipy/dipy/pull/3325/files#diff-674b7eaaf375ba4af888af6d75137f735bb8608d56d3b9c84d4d8f421b397282R1696

so leaving it.